css-positioning/fixed.css

```diff
@@ -0,0 +1,13 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+
+  /* your code here */
+  position: fixed;
+  top: 15%;
+  left: 33%;
```

Comment:
You can just set `width: 100%` and `text-align` will take care of centering this for you 👍